### DISCUSSION TITLE
2024년 7-8월 Tech Event 모음

### DISCUSSION THREAD

**Discussion by yeniful** [DeveloperAcademy-POSTECH/.github#38] [General]
# 해커톤
|행사 이름|행사 날짜|
|---|---|
|[2024 블레이버스 MVP 개발 해커톤](https://onoffmix.com/event/303086) | 2024년 8월 3일 (토) - 8월 14일 (수)|
|[해커그라운드 해커톤 2024](https://onoffmix.com/event/303111) | 2024년 8월 25일 (일) - 8월 27일 (화)|
|[SW융합 해커톤 대회 경북 참가팀 모집](https://www.ptp.or.kr/main/board/view.do?menu_idx=116&manage_idx=15&board_idx=6452)|2024년 8월 23일 (금) - 8월 25일 (일)|

# 컨퍼런스 및 세미나
|행사 이름|행사 날짜|
|---|---|
|[I/O Extended 2024 Incheon](https://festa.io/events/5477)|2024년 7월 27일 (토) 오후 01:00 - 오후 07:30|
|[HYPER-APP 2024](https://festa.io/events/5274)|2024년 8월 10일 (토) 오후 01:00 - 오후 06:30|
|[SOUTH SUMMIT: KOREA 2024 DEEPTECH & AI](https://www.southsummit.io/korea/)|2024년 9월 25일 (수) - 9월 27일 (금)|
|[제6회 테크 데이 - OpenLab 성장 스토리](https://devocean.sk.com/events/view.do?id=201)| 2024년 8월 8일 (목)|

---
이 외의 개발 행사가 있다면 댓글로 알려주세요! 업데이트 하겠습니다.
단 주최 기관과 행사 내용에 따라 게시되지 않을 수 있습니다.

**Comment by Younkyum** [comment]
우와! 행사 정보 너무 감사합니다 예니 👍🏻

**Reply by yeniful** [reply]
앞으로도 많관부 해주세요!!

**Comment by Guryss** [comment]
예니 최고! 좋은 정보 감사합니다 😊

**Reply by yeniful** [reply]
구리스 감사해용 !!!
